src/appleseed/foundation/platform/compiler.h

```diff
@@ -96,20 +96,20 @@ namespace foundation
 // Named APPLESEED_ALIGN instead of simply ALIGN to prevent a collision with the ALIGN
```

Comment:
I think it's now safe to remove the last two lines of this comment ("Named APPLESEED_ALIGN...").